api/src/pcapi/scripts/provider/bulk_create_venue_provider.py

```diff
@@ -0,0 +1,32 @@
+# This script is used to automatically create venue_provider from
+# an existing provider and multiple venues
+
+from pcapi.core.offerers import models as offerer_models
+from pcapi.core.providers import api as provider_api
+from pcapi.core.providers import models as provider_models
+from pcapi.models import db
+
+
+def create_venue_provider(provider_id: int, venue_ids: list[int]) -> None:
+    provider = provider_models.Provider.query.get(provider_id)
+    if not provider or not provider.hasOffererProvider:
+        print("Provider not found or not synchronisable")
+        return
+
+    venues = (
+        db.session.query(provider_models.Venue)
+        .outerjoin(provider_models.VenueProvider)
+        .filter(offerer_models.Venue.id.in_(venue_ids))
+        .all()
+    )
+
+    if len(venues) != len(venue_ids):
+        print("Some venues were not found")
+        return
```

Comment:
[détail] Autant afficher les identifiants inconnus : 

```suggestion
    missing = {venue_ids} - {venue.id for venue in venues}
    if missing:
        print(f"Some venues were not found: {missing}")
        return
```